tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun_test.go

```diff
@@ -1191,6 +1225,42 @@ var expectedPipelineRunIteration1 = &v1beta1.PipelineRun{
 	},
 }
 
+var expectedPipelineRunIteration1WithWhiteSpace = &v1beta1.PipelineRun{
```

Comment:
`expectedPipelineRunIteration1WithWhiteSpace` why is this called `expectedPipelineRunIteration1` i.e. 1 at the end?

Reply:
oh, that should be a mistake,  I copied it from another name.

Reply:
I corrected it and remove 1